app.py

```diff
@@ -2265,7 +2265,7 @@ def change_language():
     # Remove 'keyword_lang' from session, it will automatically be renegotiated from 'lang'
     # on the next page load.
     session.pop('keyword_lang')
-    return jsonify({'succes': 200})
+    return jsonify({'succes': 204})
```

Comment:
Why not using `make_response` her too? Since you've already changed it in other places

Reply:
Because it would give an unknown error..